tests/components/aemet/test_weather.py

```diff
@@ -51,9 +51,8 @@ async def test_aemet_weather(hass):
     assert forecast.get(ATTR_FORECAST_PRECIPITATION_PROBABILITY) == 30
     assert forecast.get(ATTR_FORECAST_TEMP) == 4
     assert forecast.get(ATTR_FORECAST_TEMP_LOW) == -4
-    assert (
-        forecast.get(ATTR_FORECAST_TIME)
-        == dt_util.parse_datetime("2021-01-10 00:00:00+00:00").isoformat()
+    assert forecast.get(ATTR_FORECAST_TIME) == dt_util.parse_datetime(
```

Comment:
We should not change the weather platform. It should still return an isoformatted UTC datetime string. It's only the sensor entities that are affected.